BD-1616: Add Canvas Flow steps limit (#4115)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/canvas/create_a_canvas/create_a_canvas.md

As you begin to add more steps, you can view your entire Canvas using either the **Detailed View** or **Simplified View**. **Simplified View** shows just the component icons as a high-level visual of your user journey whereas **Detailed View** shows the expanded step details. Depending on your preferences, you can toggle between these views!
{% endalert %}

{% alert warning %}

> A Canvas built using Canvas Flow has a maximum limit of 200 steps. If your Canvas exceeds 200 steps, loading issues will occur.

Reviewer: ```suggestion
A Canvas built using Canvas Flow can contain up to 200 steps. If your Canvas exceeds 200 steps, loading issues will occur.
```
